src/components/DrawCore/Item.tsx

```diff
@@ -3,11 +3,62 @@ import { View, Text, StyleSheet } from 'react-native';
 import { G, Line, Ellipse, Rect, Path, ForeignObject } from 'react-native-svg';
 import type { DrawItem, Point } from '../../types';
 
-const transformPointsToPath = (points: Point[]) => {
+// properties of a line
+const line = (pointA: Point, pointB: Point) => {
```

Comment:
method is duplicated in [CurrentAnimatedItem.tsx](https://github.com/ArchireportTeam/react-native-svg-draw/pull/9/files#diff-cd7f266ce63e28ad19fd2c2b1d75e00ee80bb6cfbf049099adcaef582c19623f)